app/src/main/java/com/randomappsinc/studentpicker/home/MainActivity.java

```diff
@@ -42,6 +42,9 @@
 public class MainActivity extends StandardActivity {
 
     public static final String LIST_NAME_KEY = "listName";
+    public static final int NAME_CHOOSING_FRAGMENT = 0;
```

Comment:
Some thoughts here:
1. I feel like the receiver (ListActivity) should contain the keys, not the sender (MainActivity). Putting the keys within the sender makes this functionality less flexible in the long run. If ListActivity can be started from another place in the future, we will need to define a new set of keys, which will lead to dangerous copy-paste. Think about this problem as ListActivity "exposing" an API for other places to call to control its UI/UX.
2. As mentioned in class, each component should "hide" as much of its functionality as possible. By passing in the ViewPager index indirectly, ListActivity is revealing that it uses a ViewPager. This is fragile in the future since ListActivity may move off of a ViewPager implementation. Let's just create a new key like "START_ON_EDIT_PAGE" and the value will be a true/false boolean.
3. Do the simple thing first! Right now, there isn't a need to expose such a powerful API since by default, the ViewPager starts off on index 0 and we only need to make it 1 every once in a while. A powerful API also comes with risk, since a caller might pass in 2 or something and cause index out of bounds.